aws/kms/variables.tf

```diff
@@ -57,3 +57,8 @@ variable "principal_roles" {
   type        = list(string)
   default     = null
 }
+
+variable "aws_account_id" {
+  description = "The ID of AWS account"
+  type        = string
+}
```

Comment:
Please add EOL and markdown on the key policy resource that it will overwrite default one